### DISCUSSION TITLE
Organizing the Database

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#22] [Ideas]
We are rapidly adding new ventures to the database and before long a single table is not going to be satisfactory. Eventually I will create general categories to group the businesses and they can be split further and further as each group grows. Opening this discussion to start brainstorming the groups 💭

**Comment by jtmuller5** [comment]
I'm thinking 5 related businesses will be the threshold for creating a separate category. HustleGPT loves eco friendly business so that will probably be the first.

**Reply by artseabra** [reply]
Do you know what the categories would be yet?
My hustleGPT asst. suggested I launch a dev/design assets. What would this be, `Resources`?

**Reply by Oricrypto** [reply]
Mine suggested eco friendly stuff as well

**Reply by jtmuller5** [reply]
Hey ya'll sorry for the delay, the new database has about 8 categories and the plan is to keep expanding as we have more folks join. I think home care and Social Movement might be next!
